apps/juicebox_web/test/lib/web/channels/stream_channel_test.exs

```diff
@@ -45,24 +45,24 @@ defmodule JuiceboxWeb.StreamChannelTest do
   describe "joining a channel with no existing videos" do
     setup [:setup_stream, :setup_and_join_socket]
     test "after joining it receives an empty queue", _ do
-      assert_push "remote.action", %{ videos: [], type: "QUEUE_UPDATED" }
+      assert_push "remote.action", %{videos: [], type: "QUEUE_UPDATED"}
     end
 
     test "after joining it does not receive the playing video", _ do
-      refute_push "remote.action", %{ playing: _, type: "PLAYING_CHANGED", time: _ }
+      refute_push "remote.action", %{playing: _, type: "PLAYING_CHANGED", time: _}
     end
   end
 
   describe "joining a channel with an existing stream" do
     setup [:setup_stream, :add_videos_to_stream, :setup_and_join_socket]
 
     test "after joining it receives the proper queue", %{video_2: video_2} do
-      expected_queue_response = %{ videos: [ video_2 ], type: "QUEUE_UPDATED" }
+      expected_queue_response = %{videos: [ video_2 ], type: "QUEUE_UPDATED"}
```

Comment:
There is no whitespace around parentheses/brackets most of the time, but here there is.